OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3538,91 +3545,244 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     ፪ቤት፡ ያሹሩት፡ በድሙሩ፡ ፳፩ነው፡
                                     ይህነን፡ የተጻፈውን፡ የሸጠና፡ የሰረቀ፡ የፋቀ፡ በሥልጣነ፡ ጴጥሮ<add place="above">ስ</add>፡ ወጳውሎስ፡
                                     ውጉዘ፡ ይኩንx x x
                                     </q>
                                 </item>
                                 
-                                <item xml:id="a2">
+                                <item xml:id="a2" corresp="#h2">
                                     <locus target="#5r"/>
-                                    <desc type="Inventory">An additional list of objects, separated from
+                                    <desc type="Inventory">An additional list of objects written in the same hand as <ref target="#a1"/>, separated from
                                     <ref target="#a1"/> by a fine line.</desc>
                                     <q xml:lang="am">በ፯ወቄት፡ ከ፫ድሪም፡ የጕልላት፡ መስቀል፡ የተሰራ፡ ቍጽሩ፡
                                     ፲፱ነው፡ ከቃቤት፡ አለ፡</q>
                                 </item>
                                 
                                 <item xml:id="a3">
                                     <locus target="#7r"/>
-                                    <desc type="OwnershipNote">Note written in pale black ink in the upper corner of the folio, stating that this manuscript belonged to 
+                                    <desc type="OwnershipNote">Note written in pale black ink in the upper inner corner of the folio, stating that this manuscript belonged to 
                                         <placeName ref="INS0101MadhaneAlam"/>.</desc>
                                     <q xml:lang="gez">ተአምረ፡ ማርያም፡ ዘቅዱስ፡ <placeName ref="INS0101MadhaneAlam">መድኃኔ፡ ዓለም፡</placeName></q>
                                 </item>
+                                
+                                <item xml:id="e1">
+                                    <locus target="#1r"/>
+                                    <desc>Pen trial in black ink.</desc>
+                                    <q xml:lang="gez">ፈቲነ፡ ብርዕ</q>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#1v"/>
+                                    <desc>Pen trial in black ink.</desc>
+                                    <q xml:lang="gez">ፈቲነብርዕ</q>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <locus target="#105r"/>
+                                    <desc>Indication for liturgical reading in the upper margin written by the main hand.</desc> 
+                                    <q xml:lang="gez">ዘዓርብ፡ ዘ<hi rend="rubric">፱</hi>ሰዓት፡ ድርሳን<hi rend="rubric">፨</hi></q>
+                                </item>
+                                
+                                <item xml:id="e4">
```

Comment:
I grouped all different kinds of scribal corrections under one item. Shall we brieflydisucss also this? I don't have preferences on this topic (just I feel it might get too segemented)

Reply:
"Scribal practices" is already on the agenda, I added also this question.